server/views/pages/visits/summary.njk

```diff
@@ -44,7 +44,24 @@
                 namePrefix: "date-picker",
                 hint: {
                   text: "Enter a date"
-                }
+                },
+                items: [
```

Comment:
Unfortunately the way this component works means if you change any defaults you have to explicitly specify each input item 🙁